pkgs/applications/science/biology/trf/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchFromGitHub, }:
+
+stdenv.mkDerivation rec {
+  pname = "trf";
+  version = "4.09.1";
+
+  src = fetchFromGitHub {
+    owner = "Benson-Genomics-Lab";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-73LypVqBdlRdDCblf9JNZQmS5Za8xpId4ha5GjTJHDo=";
+  };
+
+  meta = with lib; {
+    description = "Tandem Repeats Finder: a program to analyze DNA sequences";
+    homepage = "https://tandem.bu.edu/trf/trf.html";
+    license = licenses.agpl3;
```

Comment:
```suggestion
    license = licenses.agpl3Plus;
```

Reply:
I didn't know about agpl3 or later, thanks!

Reply:
That's from here https://github.com/Benson-Genomics-Lab/TRF/blob/355c1f92b24f6b2e71650eda3549d46f0316e571/src/trf.c#L7-L10: `version 3 of
the License, or (at your option) any later version`.